### PR TITLE
DM-11136: add section on calibrations within the CameraMapper

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -679,6 +679,16 @@ exists in the repository. But with a bypass function the Butler can not know
...
 derived from the `location.dataId` and do not actually need the object itself.
 
+Calibs
+^^^^^^
+
+The ``CameraMapper`` provides support for calibrations (e.g., bias, dark, flat, fringe frames),
+which are read from a separate root directory than the other datasets.
+An important requirement is that no calibration shall ever be used unexpectedly.
+In order to achieve this requirement, we read and write the calibrations from separate directories:
+calibrations shall be read from the calibration root directory, and written to the same root directory as other butled datasets.
```
Comment (n8pease) [inline index.rst]: butled->butler

**Comment by n8pease** [comment]
aside from the one typo ("butled"), it looks good.

File: index.rst (hunk 1 above)
Comment (PaulPrice) [reply index.rst]: That was quite on purpose, but OK.

Comment (n8pease) [reply index.rst]: oh. hmm. butlered?

Comment (n8pease) [reply index.rst]: buttled?

**Pull request merged**